core/urls.py

```diff
@@ -1,10 +1,18 @@
 from django.urls import include, path
 from knox import views as knox_views
 from core.views import LoginView
+from core import views as core_views
 
 urlpatterns = [
     path('auth/login/', LoginView.as_view(), name='knox_login'),
     path(r'auth/logout/', knox_views.LogoutView.as_view(), name='knox_logout'),
     path(r'auth/logoutall/', knox_views.LogoutAllView.as_view(), name='knox_logoutall'),
     path(r'auth/', include('rest_social_auth.urls_knox')),
+    path(r'rest/', include('rest_framework.urls')),
```

Comment:
As far as I can tell, this one adds login & logout urls for the DRF. Do we need those? When I logged in, I got the error below

![image](https://user-images.githubusercontent.com/5672438/53869458-5a935880-4009-11e9-8326-b884c10bab5e.png)